src/main/java/net/minecraftforge/fluids/FluidUtil.java

```diff
@@ -423,21 +425,62 @@ else if (block instanceof BlockLiquid)
     }
 
     /**
+     * Gets the fluid handler for a given fluid's block.
+     * The handler's fill method will place a fluid block with the appropriate amount when not simulated.
+     *
+     * @param world the world the block has to be placed in
+     * @param pos   the position the block has to be placed at
+     * @param fluid the fluid to place
+     * @return the fluid block's fluider handler
+     */
+    private static IFluidHandler getPlacementFluidHandler(final World world, final BlockPos pos, Fluid fluid)
+    {
+        final Block block = fluid.getBlock();
+        IFluidHandler handler;
+        if (block instanceof IFluidBlock)
+        {
+            handler = new FluidBlockWrapper((IFluidBlock) block, world, pos);
+        }
+        else if (block instanceof BlockLiquid)
+        {
+            handler = new BlockLiquidWrapper((BlockLiquid) block, world, pos);
+        }
+        else
+        {
+            handler = new VoidFluidHandler()
+            {
+                @Override
+                public int fill(FluidStack resource, boolean doFill)
+                {
+                    if (doFill)
+                    {
+                        world.setBlockState(pos, block.getDefaultState(), 3);
+                    }
+                    return super.fill(resource, doFill);
+                }
+            };
+        }
+        return handler;
+    }
+
+    /**
      * Tries to place a fluid in the world in block form.
      * Makes a fluid emptying sound when successful.
      * Checks if water-like fluids should vaporize like in the nether.
      *
      * Modeled after {@link net.minecraft.item.ItemBucket#tryPlaceContainedLiquid(EntityPlayer, World, BlockPos)}
      *
      * @param player     Player who places the fluid. May be null for blocks like dispensers.
-     * @param worldIn    World to place the fluid in
+     * @param world      World to place the fluid in
      * @param fluidStack The fluidStack to place.
      * @param pos        The position in the world to place the fluid block
      * @return true if successful
+     * @deprecated use {@link #tryPlaceFluid(EntityPlayer, World, BlockPos, ItemStack, FluidStack)}
      */
-    public static boolean tryPlaceFluid(@Nullable EntityPlayer player, World worldIn, FluidStack fluidStack, BlockPos pos)
+    @Deprecated
+    public static boolean tryPlaceFluid(@Nullable EntityPlayer player, World world, FluidStack fluidStack, BlockPos pos)
```

Comment:
Please don't change parameter names, it just makes the diff harder to read and understand.